### ISSUE TITLE
Support for Nginx

### ISSUE THREAD

**Issue by gogl92** [Tegomena/yii-dev#2]
This version with apache is working fine but Nginx support could bring advantages:

I created [this](https://gist.github.com/gogl92/e018d4ffc974b65c2ca7a1af78b57059) script to automate the server configuration, the package installation example could be used as example to add Nginx support and configuration.

**Comment by Tegomena** [comment]
Thanks @gogl92 - May I "cannibalize" your script? It would be necessary to have parts of it in the Dockerfile and other parts in an extra script

**Comment by ljfreelancer88** [comment]
Steps also on how to edit configuration of nginx too. The good steps is the Yii example on their documentation.

**Comment by gogl92** [comment]
Sure, completely open source, also any improvement is welcome

**Comment by Tegomena** [comment]
Hhhm, I tried to get Nginx and php-fpm in the same Docker container up and running, but thats difficult. Would require at least two containers (one for Nginx, one for PHP). This can easily be done via docker-compose, but then we are loosing the "easy" way of starting up the whole thing just with one container.

What do you think?

**Comment by gogl92** [comment]
I think that this will require two containers and unfortunately will make it a little more complex

**Comment by Tegomena** [comment]
Yes, indeed. But on the other hand, if we  want an easy possibility to try out other databases etc., these things also need to be done via docker-compose. I'll try it out.
